impl/config/config.go

```diff
@@ -81,6 +85,9 @@ func GetDefaultConfig() Config {
 			CacheTTLSeconds:  600,
 			CacheSizeLimitMB: 500,
 		},
+		Log: LogConfig{
+			Level: logrus.InfoLevel.String(),
```

Comment:
default path?

Reply:
Default is to not log to a file, just stdout. Logging to file seems like an edge case - most things that are running this will have their own way of handling logging and will just expect the program to write to stdout.

Reply:
agree, sgtm. might be good to doc that somewhere
consider a new `config.md` in a `docs` folder that goes into stuff like this, and DB stuff - can do this in another PR if easier